sql/moz-fx-data-shared-prod/account_ecosystem_derived/ecosystem_user_id_lookup_v1/metadata.yaml

```diff
@@ -17,11 +17,9 @@ scheduling:
   dag_name: bqetl_account_ecosystem
   depends_on_past: true
   referenced_tables:
-    - [
-        "moz-fx-data-shared-prod",
-        "firefox_accounts_stable",
-        "account_ecosystem_v1",
-      ]
+    - - "moz-fx-data-shared-prod"
+      - "firefox_accounts_stable"
+      - "account_ecosystem_v1"
```

Comment:
Seeing this syntax here rather than `[]` is surprising to me.

Reply:
It was unexpected that yamllint didn't agree with the original indent here. I chose somewhat arbitrarily based on personal preference between this and the other option that passed yamllint indent checks, which is:
```diff
-    - - "moz-fx-data-shared-prod"
-      - "firefox_accounts_stable"
-      - "account_ecosystem_v1"
+    - [
+      "moz-fx-data-shared-prod",
+      "firefox_accounts_stable",
+      "account_ecosystem_v1"
+    ]
```

Two advantages of my choice are that it's shorter, and using `ruamel.yaml` in round-trip mode doesn't force it onto a single line.

Advantages of the alternative include preserving the existing `[]` notation, and that the separation of top-level list items is slightly more clear visually.